Cargo.toml

```diff
@@ -645,6 +645,106 @@ zstd = "0.13.3"
 opt-level = 3
 
 [patch.crates-io]
+# The following entries are auto-generated by /bin/bash
```

Comment:
Ignore this. This will disappear when https://github.com/anza-xyz/solana-sdk/pull/74 is released.